gammapy/cube/tests/test_simulate.py

```diff
@@ -50,6 +52,49 @@ def dataset():
     return dataset
 
 
+@pytest.fixture(scope="session")
```

Comment:
Please avoid the copy and pasted code here and just add the case with the background model only to one of the existing tests...

Reply:
Ok.